includes/class-activity-dispatcher.php

```diff
@@ -51,7 +52,8 @@ public static function send_update_activity( $activitypub_post ) {
 		$activitypub_activity->from_post( $activitypub_post->to_array() );
 
 		foreach ( \Activitypub\get_follower_inboxes( $user_id ) as $inbox => $to ) {
-			$activitypub_activity->set_to( $to );
+			$activitypub_activity->set_to( 'https://www.w3.org/ns/activitystreams#Public' );
+			$activitypub_activity->set_cc( $to );
```

Comment:
`$to` includes only the followers that are on the same server as `$inbox`. Is that correct or should we add the complete followers list?

Reply:
I think this is correct, the for loop will take care so that we get to all followers per server. 

We could add the user followers collection endpoint in `to` making it an array with the special Public address. But imho it is not necessary for now